unity-client/Assets/Scripts/MainScripts/DCL/Controllers/MapRenderer/GlobalUsersPositionMarker/Handlers/FetchScenesHandler.cs

```diff
@@ -0,0 +1,121 @@
+﻿using System;
+using System.Collections.Generic;
+using UnityEngine;
+using DCL.Interface;
+using System.Collections;
+using UpdateMode = MapGlobalUsersPositionMarkerController.UpdateMode;
+
+internal class FetchScenesHandler : IDisposable
```

Comment:
Could we add a documentation header here to explain what is the responsibility of this class? Thanks!